RELEASE.rst

```diff
@@ -1,6 +1,20 @@
 Release Notes
 -------------
 
+Version 0.2.0
+=============
+
+- Implemented authorization for taxonomy pieces of REST API.
+- Styled full text search box per the designer.
+- Updated REST API docs.
+- Imported static assets during course import.
+- Added abstract base model to contain audit fields.
+- Added faceted search for taxonomy items.
+- Added Additional APIs for roles.
+- Switched to using get_perms for cleaner code.
+- Added JavaScript infrastructure to run unit tests.
+- Added slide out panel for learning resource details.
+- Styled left hand navigation pane with glyphs.
```

Comment:
My concern with this, is that makes the release notes unable to be automated.  The list you see is a copy of the merged feature commit messages.  I think that probably means we want better commit messages going forward, but it will make releases much easier and faster if we don't have to manually rewrite the messages. @pdpinch ?